### DISCUSSION TITLE
Make the default theme as a better example

### DISCUSSION THREAD

**Discussion by quank123wip** [Milkdown/milkdown#420] [Ideas]
As you can see, the color variables in default theme now is not very "semantic", and have no doc to imply how they affect the default interface. Although devs can just fire up a instance and inspect their changes to figure out, and I have no difficulties on this, but seems a better theme doc and naming would help others.

**Comment by Saul-Mirone** [comment]
How do you think we can improve this, by providing a better doc?

**Reply by quank123wip** [reply]
Yes, maybe. A good live example is good, too. Like a editor instance with color palattes aside, maybe like [FAST](https://color.fast.design) do.
